### PR TITLE
Friday afternoon doc-work

### PR DESCRIPTION
**Pull request by asgardiator**
In order of visibility: 

1. Rewrite `docs/web.md` introcudtion to explain better the information within Nock/Hoon/Arvo.
  1. `%s/aspie/academic/g`
  1. Intro structure mirrors Urbit structure: style points
1. Some minor formatting in `/arvo/basic.md`
  1. a blockquote explaining `*@`, which was used in the example.
  1. Pronunciation guides for digraphs.
1. File == directory section for `/using/web`, along with the easter egg hint that editing `web.md` allows you to edit your default page at `some-ship.urbit.org`.

### CODE REVIEW

File: docs/using/web.md [hunk 1]
```diff
@@ -55,6 +55,10 @@ To view your file as raw `html`:
 
     http://localhost:8080/test.html
 
+You may have noticed that we just created both `test.md` and `test/`. In urbit, there is no distinction between a directory and a file. So, if you're adding files directly from unix, simply `mkdir test` and urbit will treat the file and directory as the same entity. 
+
+You can use this feature to change your homepage at `your-urbit.urbit.org`. By default, this page will display some info about your urbit, but you can create the file `/home/web.md` to customize this page to your heart's content.
```
Comment (galenwp) [inline docs/using/web.md]: This is great!  I had been meaning to put this in.  

`/home/web.md` actually exists by default, so could be worth rephrasing to that end.  

When we were talking earlier, I think I sort of lost the thread when I was trying to get at this exact point.  That you can customize the homepage, and pretty easily turn it into a personal talk feed, static site, or whatever.  Even the identity component can be customized.  I had intended to make this one of the children of `/docs/using/web/` (like http://urbit.org/docs/using/web/static/) but never got around to it.

**Comment by galenwp** [comment]
This looks great to me — I'd just rephrase slightly since `web.md` exists by default.  (Here's mine: http://talsur-todres.urbit.org/.md)

**Comment by asgardiator** [comment]
Improved suggested phrasing, with a few urbit -> Urbit capitalizations for flair.

**Comment by galenwp** [comment]
Thanks!

**Comment by philipcmonk** [comment]
Since cleanorg has been merged into master, would anyone object to me deleting the cleanorg branch?

**Comment by galenwp** [comment]
Yeah, get rid of it. Thanks!

**Pull request merged**